[2025-09-10T00:39:14.582221] [INFO] [SYSTEM] Start new run (run_id=2efb3288-b8fe-4807-90e6-82e94634811a)
[2025-09-10T00:39:37.780326] [INFO] [GENERAL] debug_utils.py:log_debug:238 - User int input valid: 3
[2025-09-10T00:39:48.398791] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Counts computed for kit build.
[2025-09-10T00:39:52.979311] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Base64 length bounds computed for pad selection.
[2025-09-10T00:40:00.170198] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Using PAD size (bucketized): 128
[2025-09-10T00:40:02.962364] [INFO] [GENERAL] debug_utils.py:log_debug:238 - User int input valid: 1
[2025-09-10T00:40:04.028151] [INFO] [GENERAL] debug_utils.py:log_debug:238 - User int input valid: 1
[2025-09-10T00:40:05.260900] [INFO] [GENERAL] debug_utils.py:log_debug:238 - User int input valid: 1
[2025-09-10T00:40:05.261899] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Argon2id parameters confirmed for kit.
[2025-09-10T00:40:05.261899] [WARNING] [CRYPTO] debug_utils.py:log_debug:238 - Combinatorial hardness below policy threshold.
[2025-09-10T00:40:07.147566] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Combinatorial hardness evaluation complete.
[2025-09-10T00:40:07.149558] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() via node-bridge
[2025-09-10T00:40:07.455186] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() complete.
[2025-09-10T00:40:07.456190] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() via node-bridge
[2025-09-10T00:40:07.522213] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() complete.
[2025-09-10T00:40:07.524215] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() via node-bridge
[2025-09-10T00:40:08.102607] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() complete.
[2025-09-10T00:40:08.103608] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() via node-bridge
[2025-09-10T00:40:08.690811] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - sss_split() complete.
[2025-09-10T00:40:10.308981] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Using user-provided password='Rex' (raw).
[2025-09-10T00:40:10.308981] [INFO] [CRYPTO] debug_utils.py:log_debug:238 - Starting Argon2id KDF (Noble). pass='Rex', salt(b64)='O3o2lAl/gyK0QY0dhYcR6A=='
[2025-09-10T00:40:10.411516] [ERROR] [GENERAL] debug_utils.py:log_exception:286 - Error during encrypted kit preparation
